pkg/registrationserver/registrationserver.go

```diff
@@ -0,0 +1,126 @@
+package registrationserver
+
+import (
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	errs "github.com/pkg/errors"
+)
+
+var (
+	// Commit current build commit set by build script
+	Commit = "0"
+	// BuildTime set by build script in ISO 8601 (UTC) format:
+	// YYYY-MM-DDThh:mm:ssTZD (see https://www.w3.org/TR/NOTE-datetime for
+	// details)
+	BuildTime = "0"
+	// StartTime in ISO 8601 (UTC) format
+	StartTime = time.Now().UTC().Format("2006-01-02T15:04:05Z")
+)
+
+// RegistrationServer bundles configuration, logging, and HTTP server objects in a single
+// location.
+type RegistrationServer struct {
+	config     *configuration.Registry
+	router     *mux.Router
+	httpServer *http.Server
+
+	logger      *log.Logger
+	routesSetup sync.Once
+}
+
+// New creates a new RegistrationServer object with reasonable defaults.
+func New(configFilePath string) (*RegistrationServer, error) {
+	srv := &RegistrationServer{
+		router: mux.NewRouter(),
+		logger: log.New(os.Stderr, "", 0),
```

Comment:
do we need to log to `Stderr`?

Reply:
Good question. What is the best practice for a k8s application? Logging to stdout?